mod_reforged/hooks/skills/effects/bleeding_effect.nut

```diff
@@ -0,0 +1,131 @@
+::Reforged.HooksMod.hook("scripts/skills/effects/bleeding_effect", function(q) {
+	q.m.Stacks <- 1;
+	q.m.StacksThisTurn <- 1;
+	q.m.StacksForMoraleCheck <- 3;
+	q.m.MaxMalusMult <- 0.5;
+	q.m.MalusMultPerStack <- 0.05;
+
+	q.create = @(__original) function()
+	{
+		__original();
+		this.m.Description = "This character is bleeding profusely and will lose hitpoints each turn while acting slower and losing courage.";
+		this.m.IsStacking = false;
+	}
+
+	q.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.push({
+			id = 10,
+			type = "text",
+			icon = "ui/icons/regular_damage.png",
+			text = ::Reforged.Mod.Tooltips.parseString("Receive " + ::MSU.Text.colorRed(this.getDamage()) + " damage per [turn|Concept.Turn]")
+		});
+		ret.push({
+			id = 11,
+			type = "text",
+			icon = "ui/icons/bravery.png",
+			text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeMult(this.getMalusMult()) + " reduced [Resolve|Concept.Bravery]")
+		});
+		ret.push({
+			id = 12,
+			type = "text",
+			icon = "ui/icons/initiative.png",
+			text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeMult(this.getMalusMult()) + " reduced [Initiative|Concept.Initiative]")
+		});
+		ret.push({
+			id = 13,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = ::Reforged.Mod.Tooltips.parseString("If at least " + ::MSU.Text.colorRed(this.m.StacksForMoraleCheck) + " stacks are received in a single [turn|Concept.Turn], a negative [morale check|Concept.Morale] is immediately triggered")
```

Comment:
semicolon missing

Reply:
This one is correct because it's a table.